Gemfile

```diff
@@ -7,7 +7,7 @@ gemspec path: File.expand_path('..', __FILE__)
 gem 'byebug' unless ENV['TRAVIS']
 gem 'pry-byebug' unless ENV['CI']
 
-gem 'activemodel', ENV['RAILS_VERSION'] if ENV['RAILS_VERSION']
+gem 'rails', ENV['RAILS_VERSION'] if ENV['RAILS_VERSION']
```

Comment:
ActiveFedora should not depend on all of Rails. We have a number of applications that are just ETL with ActiveFedora.

Reply:
That was your change.  I think you had done that in the Gemfile here just for testing purposes since a really old version of rails was getting pulled in.  I'll try rolling it back and see if it works with the pinning of bundler.

Reply:
Sweet!  That worked!

Reply:
Ugh, sorry. Thanks for fixing it.